pkg/server/server.go

```diff
@@ -87,14 +90,22 @@ func NewServer(config *viper.Viper) *Server {
 
 // Serve runs the server
 func Serve(config *viper.Viper) {
+	beeline.Init(beeline.Config{
+		WriteKey: os.Getenv("HNY_KEY"),
+		Dataset:  "easi-app",
+		STDOUT:   (os.Getenv("HNY_KEY") == ""),
```

Comment:
If there is no Honeycomb key passed in, then this instruction says to emit all the individual spans/logs/events to standard out, which can be useful for local debugging.